### PR TITLE
Correctly report E501 when the first line of a docstring is too long

### LINKED ISSUE #622: max line length not enforced for long first line docstrings
Somewhat new to python tooling and pycodestyle, but I believe this should trigger the (default) max line length E501:

```
def foo():
    """Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis pulvinar vitae

    Lorem ipsum dolor sit amet, consectetur adipiscing elit.
    """
```

The first docstring line is 83 characters (> 79). I believe pycodestyle may be looking at the tokenized docstring, which doesn't include the leading whitespace in front of the opening """ in the physical line.

### PR DESCRIPTION
**Pull request by asottile**
Resolves #622

### CODE REVIEW

File: pycodestyle.py [hunk 1]
```diff
@@ -1199,7 +1199,7 @@ def comparison_type(logical_line, noqa):
 
 
 def bare_except(logical_line, noqa):
-    r"""When catching exceptions, mention specific exceptions whenever possible.
+    r"""When catching exceptions, mention specific exceptions when possible.
```
Comment (asottile) [inline pycodestyle.py]: this line was too long, I tried shortening it without changing meaning

**Comment by asottile** [comment]
@sigmavirus24  thoughts on this?

**Comment by asottile** [comment]
@sigmavirus24 bump again :)

**Comment by sigmavirus24** [comment]
Sorry for the delay @asottile. I've been quite busy for a while now.

**Pull request merged**